src/components/GameLobby.jsx

```diff
@@ -5,46 +5,79 @@ import SetUser from "./SetUser";
 import establishIoConnection from "../lib/socket/";
 import SocketEvents from "../lib/enums/socketEvents";
 import GameBoard from "./GameBoard";
+import WinnerPage from "./WinnerPage";
 
 function GameLobby() {
   const [lobbyName] = useState(useParams().lobbyName);
   const [gameStatus, updateGameStatus] = useState();
   const [gameData, updateGameData] = useState();
   const [initialUserState, updateInitialUserState] = useState();
   const [userName, setUserName] = useState();
+  const [users, updateUserData] = useState({});
```

Comment:
if you're creating a user state in this component it's no longer necessary to also create that a second time in the child component, you can just pass down the user and updateUserData function and move the state up like you would with class components